add-chat-push-notifications/Podfile

```diff
@@ -4,15 +4,19 @@ platform :ios, '13.0'
 target 'SwiftPushTest' do
   # Comment the next line if you don't want to use dynamic frameworks
   use_frameworks!
-pod 'AzureCommunicationChat', '1.3.0-beta.1'
+
+pod 'AzureCommunicationChat', '~> 1.3.0'
```

Comment:
This should probably be `~> 1.3` to allow it to pick up patch versions.

Reply:
agree. Change pushed.